client/lib/posts/actions.js

```diff
@@ -325,7 +327,7 @@ PostActions = {
 			return;
 		}
 
-		changedAttributes = normalizeApiAttributes( changedAttributes );
+		changedAttributes = PostActions.normalizeApiAttributes( changedAttributes );
```

Comment:
I refactored that part, but I have feeling that it still could be done better :)